CONTRIBUTING.rst

```diff
@@ -71,3 +71,11 @@ Notebook contributions
   Use `nbencdec <https://github.com/ttung/nbencdec>`_ to generate the corresponding `.py` file.
 - The `.py` files allow refactor commands in the codebase to find code in the `.py` files,
   which is an important to keep the notebooks working as starfish evolves.
+
+
+Project Tracking
+-----------------
+- The starfish team uses `zenhub <https://app.zenhub.com/workspaces/starfish-dev-5b4e05b4c93e4717b2160fdb/board>`_ to track the team's progress
+and upcoming engineering raodmap.
+- If you would like to see the team's board in github and the correct rendering of epic issues it's recommended that you download the chrome extension
```

Comment:
You should point to this page instead, as it has both the Chrome and FF extensions (https://www.zenhub.com/extension).